developerGuide.t2t

```diff
@@ -257,31 +257,30 @@ From anywhere, you can now press NVDA+shift+v to have NVDA's version spoken and
 # Developer guide example 2
 
 import globalPluginHandler
+from scriptHandler import script
 import ui
 import versionInfo
 
 class GlobalPlugin(globalPluginHandler.GlobalPlugin):
 
+	@scriptHandler.script(gesture="kb:NVDA+shift+v")
 	def script_announceNVDAVersion(self, gesture):
 		ui.message(versionInfo.version)
 
-	__gestures={
-		"kb:NVDA+shift+v": "announceNVDAVersion",
-	}
-
 --- end ---
 ```
 
 This Global Plugin file starts with two comment lines, which describe what the file is for.
 
 It then imports the globalPluginHandler module, so that the Global Plugin has access to the base GlobalPlugin class.
 
-It also imports a few other modules, namely ui and versionInfo, which this specific plugin needs in order for it to perform the necessary actions to announce the version.
+It also imports a few other modules, namely ui, versionInfo and scriptHandler, which this specific plugin needs in order for it to perform the necessary actions to announce the version.
 
 Next, it defines a class called GlobalPlugin, which is inherited from globalPluginHandler.GlobalPlugin.
 
 Inside this class, it defines 1 or more events, scripts or gesture bindings.
-In this example, it defines a script method that performs the version announcement, and provides a binding from NVDA+shift+v to this script.
+In this example, it defines a script method that performs the version announcement.
+The script function from the scriptHandler module is used to assign the NVDA+shift+v shortcut to this script.

```

Comment:
You say "script function" here, but this might be confused with "script method" before that. I'd change this to "script decorator". We do say that some understanding of Python is a prerequisite. However, if you're worried about this, you could refer to the later section which explains this in more detail.